resync/workflows/statistical_analysis.py

```
@@ -0,0 +1,637 @@
+
+    residuals = [y - (intercept + slope * x) for x, y in zip(x_values, y_values)]
+    sse = sum(r**2 for r in residuals)
+    dof = max(1, n - 2)
+    residual_std = (sse / dof) ** 0.5
+
+    return intercept, slope, residual_std
+
+
+def calculate_confidence_interval(
+    slope: float,
+    intercept: float,
```

Ruff (0.15.0)

[warning] Unused function argument: `intercept`

(ARG001)